src/main/java/frc/robot/RobotContainer.java

```diff
@@ -23,11 +30,38 @@
  * subsystems, commands, and trigger mappings) should be declared here.
  */
 public class RobotContainer {
+
   // The robot's subsystems are defined here...
   private final Drivetrain m_drive = new Drivetrain();
 
   // Shuffleboard stuff
   SendableChooser<Command> m_autoCommand = new SendableChooser<>();
+  SendableChooser<Teams> m_teamChooser = new SendableChooser<>();
```

Comment:
I think you can use Driverstation.getAlliance() or something like that to get the alliance. That will simplify a bit.